src/single-spa-react.js

```diff
@@ -150,8 +149,28 @@ function chooseDomElementGetter(opts, props) {
     return () => props.domElement
   } else if (props.domElementGetter) {
     return props.domElementGetter
-  } else {
+  } else if (opts.domElementGetter) {
     return opts.domElementGetter
+  } else {
+    return defaultDomElementGetter(props)
+  }
+}
+
+function defaultDomElementGetter(props) {
```

Comment:
This is essentially a duplicate of https://github.com/CanopyTax/single-spa-angular/blob/master/src/single-spa-angular.ts#L149, but the cost of DRY and reusing was too high for me. We bundle these helper libraries into self contained UMD bundles with no deps, and changing the build process to get a shared NPM package into it was more work than it was worth to me.